x-pack/plugins/security_solution/public/detections/components/endpoint_responder/use_responder_action_data.ts

```diff
@@ -93,7 +93,22 @@ export const useResponderActionData = ({
   const [isDisabled, tooltip]: [disabled: boolean, tooltip: ReactNode] = useMemo(() => {
     // v8.13 disabled for third-party agent alerts if the feature flag is not enabled
     if (!isEndpointHost) {
-      return [isSentinelOneHost ? !isSentinelOneV1Enabled : true, undefined];
+      switch (agentType) {
+        case 'sentinel_one':
+          // Disable it if feature flag is disabled
+          if (!isSentinelOneV1Enabled) {
+            return [true, undefined];
+          }
+          // Event must have the property that identifies the agent id
+          if (!getSentinelOneAgentId(eventData ?? null)) {
```

Comment:
In this case its needed ecause `getSentineOneAgentId()` does not accept `undefined` and `eventData` could be `undefined` because that prop is optional

Reply:
oh right, I now see that eventData in this function can be `undefined`. This is good as it is then. 🚀